src/app/backend/handler/apihandler.go

```diff
@@ -555,6 +556,16 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
 			To(apiHandler.handleLogFile).
 			Writes(logs.LogDetails{}))
 
+	apiV1Ws.Route(
+		apiV1Ws.GET("/overview/").
+			To(apiHandler.handleOverview).
+			Writes(overview.Overview{}))
+
+	apiV1Ws.Route(
```

Comment:
It was removed. Why do you add this? You might solve merge conflicts wrongly, otherwise please explain.